### PR TITLE
Fix: spawnpoint in gathered data

### PR DESCRIPTION
**Pull request by sohje**
## Short Description:
Fix issues with incorrect spawnpoint_id in gathered data

### CODE REVIEW

**Comment by YvesHenri** [comment]
Nice.

File: pokemongo_bot/cell_workers/move_to_map_pokemon.py [hunk 1]
```diff
@@ -111,8 +111,8 @@ def pokemons_parser(self, pokemon_list):
         for pokemon in pokemon_list:
             disappear = int(pokemon.get('expiration_timestamp_ms', 0) / 1000) or int(pokemon.get('disappear_time', 0) / 1000)
 
-            pokemon['encounter_id'] = pokemon['encounter_id']
-            pokemon['spawn_point_id'] = pokemon['spawn_point_id']
+            pokemon['encounter_id'] = pokemon.get('encounter_id', '')
+            pokemon['spawn_point_id'] = pokemon.get('spawn_point_id', '') or pokemon.get('spawnpoint_id', '')
```
Comment (Gobberwart) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: Why two different attrs ("spawnpoint_id" or "spawn_point_id")? Does it vary or just don't know?

Comment (sohje) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: From pokemap its 'spawnpoint_id', from social - 'spawn_point_id'.
Dunno, mb in the future we will need them or remove completely.

Comment (Gobberwart) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: OK just wondering. Approving.

Comment (YvesHenri) [inline pokemongo_bot/cell_workers/move_to_map_pokemon.py]: I'm not sure on social though, but I'm planning to enable dynamic param name mapping.

**Pull request merged**